src/sgame/botlib/bot_nav_edit.cpp

```diff
@@ -68,12 +70,16 @@ bool GetPointPointedTo( NavData_t *nav, rVec &p )
 	             CONTENTS_SOLID | CONTENTS_PLAYERCLIP, 0, traceType_t::TT_AABB );
 
 	pos = qVec( trace.endpos );
-	if ( dtStatusFailed( nav->query->findNearestPoly( pos, extents, &nav->filter, &nearRef, p ) ) )
-	{
-		return false;
-	}
+	return !dtStatusFailed( nav->query->findNearestPoly( pos, extents, &nav->filter, &nearRef, p ) );
+}
 
-	return true;
+bool GetFeetPoint( NavData_t *nav, rVec &p, gentity_t *self )
+{
+	rVec extents;
+	VectorSet( extents, 640, 96, 640 );
```

Comment:
Minor enough, but why isn't Z the only coord that gets extended?

Reply:
This function code is `bool GetPointPointedTo( NavData_t *nav, rVec &p )` that I duplicated and adapted to my needs. I don't know where the values come from, otherwise I would have documented it.